src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -114,6 +115,18 @@ function IOURequestStepConfirmation({
     const isPolicyExpenseChat = useMemo(() => ReportUtils.isPolicyExpenseChat(ReportUtils.getRootParentReport(report)), [report]);
     const formHasBeenSubmitted = useRef(false);
 
+    useEffect(() => {
+        if (transaction?.participants?.findIndex((participant) => participant.accountID === payeePersonalDetails?.accountID) !== -1 || iouType !== CONST.IOU.TYPE.SPLIT) {
+            return;
+        }
+
+        const payeeParticipant = OptionsListUtils.getParticipantsOption({accountID: payeePersonalDetails?.accountID, selected: true}, personalDetails);
+        IOU.setMoneyRequestParticipants_temporaryForRefactor(transaction.transactionID, [...(transaction?.participants ?? []), payeeParticipant]);
+
+        // We only want to run it when the component is mounted
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
     useEffect(() => {
         const policyExpenseChat = participants?.find((participant) => participant.isPolicyExpenseChat);
         if (policyExpenseChat?.policyID) {
```

Comment:
@nkdengineer if the participant is a workspace chat, we shouldn't show the participant selector menu item.

Reply:
I updated to fix this case.